tools/tertiary-analysis/cell-types-analysis/ct_downsample_cells.xml

```diff
@@ -0,0 +1,60 @@
+<tool id="ct_downsample_cells" name="Cell types - Downsample Cells" version="@TOOL_VERSION@+galaxy0" profile="@PROFILE@">
+    <description>to avoid memory issues</description>
+    <macros>
+        <import>ct_macros.xml</import>
+    </macros>
+    <expand macro="requirements" />
+    <command detect_errors="exit_code"><![CDATA[
+        ln -s "${matrix}" matrix.mtx &&
+        ln -s "${genes}" genes.tsv &&
+        ln -s "${barcodes}" barcodes.tsv &&
+        downsample_cells.R --expression-data . --metadata "${metadata}" --cell-id-field "${cell_id_field}" --cell-type-field "${cell_type_field}" --array-size-limit "${array_size_limit}" --output-dir "${output_dir}" --metadata-upd "${metadata_upd}"
+
+        #if $exclusions
+        --exclusions "${exclusions}"
+        #end if
+
+         ]]></command>
+    <inputs>
+        <param name="matrix" type="data" format="txt" label="Expression matrix in sparse matrix format (.mtx)"/>
+        <param name="genes" type="data" format="tsv,tabular" label="Gene table"/>
+        <param name="barcodes" type="data" format="tsv,tabular" label="Barcode/cell table"/>
+        <param type="data" name="metadata" format="txt" label="Metadata Table" help="Metadata file mapping cells to cell types" />
+        <param type="text" name="cell_id_field" label="Cell ID Field" optional="true" value="id" help="Cell ID field">
+            <expand macro="sanitize_strings" />
+        </param>
+        <param type="text" name="cell_type_field" label="Cell Type Field" optional="true" value="inferred cell type" help="Name of cell type column in metada file">
+            <expand macro="sanitize_strings" />
+        </param>
+        <param type="data" name="exclusions" label="Exclusions file" format="yml" optional="true" help="YAML file with trivial terms and unlabelled cells" />
+        <param type="integer" name="array_size_limit" value='2000000000' label="Array Size Limit" help="Maximum length of R array" />
```

Comment:
What is this input intending to do? Could you please clarify its purpose in the in-line help?